src/components/orderstatus.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import data from "../sampleJSON/orderDetails.json";
+import backButton from "../assets/right_arrow.svg";
+import { Dialog } from "primereact/dialog";
+import { Button } from "primereact/button";
+import Timeline from "./Common/orderTimeline";
+import CustomButton from "./Button";
+import CustomImage from "./common/customimage";
+
+interface OrderStatusProps {
+  //  props
+}
+
+const OrderStatus: React.FC<OrderStatusProps> = (props) => {
+  const orderMap = data.orderMap;
```

Comment:
Add and useEffect and make the api call to the correct api.
Error will be thrown.
Catch the error and there add the hardcoded response.